nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/service/CommentService.java

```diff
@@ -87,7 +87,9 @@ public CommentServiceConfig getConfig() {
     @Override
     @SuppressWarnings("unchecked")
     public <T> T getAdapter(Class<T> adapter) {
-        if (commentManager == null) {
+        if (CommentService.class == adapter) {
```

Comment:
`adapter == CommentService.class` reads more naturally. Note that in other places you use `adapter == this.getClass()` so you should be consistent.

Reply:
I end up by choosing _myClass.class_ syntaxe (@troger) to be consistency with the different implementations of getAdapter method and in the same think i change to use adapter.cast(this) syntaxe instead by _(T) this_ although it will be end by a double check :) (Class#cast is already check before doing the cast to avoid the classCastException)